apps/gcsweb.k8s.io/certificate.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: networking.gke.io/v1
 kind: ManagedCertificate
 metadata:
   name: gcsweb
+  namespace: gcsweb
```

Comment:
Still researching if this is actually in the `gcsweb` namespace, or if it lives in `default` for some reason. I may need to work on getting more access into the aaa cluster to see this.

Reply:
@jimdaga I checked : 

```shell
ameukam@cloudshell:~ (kubernetes-public)$ kubectl -n gcsweb get managedcertificate
NAME     AGE   STATUS
gcsweb   40d   Active
```

Reply:
Great, thanks for checking that @ameukam!